app/assets/locales/messages_ccodk_default.txt

```diff
@@ -12,13 +12,11 @@ updates.check.begin=Checking for updates...
 updates.check.failed=Failed to download updates
 updates.check.start=Recheck
 updates.check.cancel=Stop checking
-updates.check.install=Install
 updates.check.cancelling=Cancelling check for updates
 updates.install.finished=App update was successful
 updates.check.network_unavailable=No network connectivity
-updates.staged.version=Update to version ${0}
+updates.staged.version=Logout and update to ${0}

```

Comment:
I left out 'version' because I didn't want it to be too long. I can add it back if that seems confusing